assets/airports/ksea.json

```diff
@@ -384,9 +384,9 @@
                 ["YVR*"]
             ]
         },
-        "SEATL5": {
-            "icao": "SEATL5",
-            "name": "Seattle Five",
+        "SEA6": {
```

Comment:
Thank god they changed this identifier, `SEATL5` was _super_ stupid and nonconformist. Bunch of hipsters... then again, _Seattle_... 🤔